Cargo.toml

```diff
@@ -39,8 +54,7 @@ incremental = true
 codegen-units = 256
 
 [patch.crates-io]
-spirv-std = { path = "./crates/spirv-std" }
-spirv-std-macros = { path = "./crates/spirv-std/macros" }
 # HACK(eddyb) needed to include this PR (until it's part of a release):
 # * Manishearth/compiletest-rs#249 (for `// only-*` support)
+# [shesp] The commit is part of 0.8 release. We can remove this once we upgrade `compiletest`.
```

Comment:
Clarification (for everyone else): this is literally just `cargo upgrade compiletest`, it should require no other changes.